modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -1031,7 +1031,7 @@ void EditorSceneImporterAssimp::_generate_node_bone_parents(const aiScene *p_sce
 				if (bone_parent_name == p_mi->get_parent()->get_name()) {
 					break;
 				}
-				if (bone_node_parent->mParent == p_scene->mRootNode) {
+				if (bone_node_parent == p_scene->mRootNode) {
```

Comment:
This causes the armature node to be included as bone. Not sure if this is a good or bad idea, but it allows the armature animations to move the entire mesh.